src/partest/scala/tools/partest/ReplTest.scala

```diff
@@ -70,7 +70,7 @@ trait Lambdaless extends ReplTest {
   }
 }
 object Lambdaless {
-  private val lambdaless = """\$Lambda\$\d+/\d+(@[a-fA-F0-9]+)?""".r
+  private val lambdaless = """\$Lambda\$\d+/(?:0x[a-f0-9]{16}|\d+)(@[a-fA-F0-9]+)?""".r
```

Comment:
You might wonder if there is a character class for that: `\p{XDigit}`

Reply:
It's also, :trollface:, conveniently 1 character shorter than its substitute `[0-9a-fA-F]`!! 😄